### PR TITLE
bug: Adding group selection on mount if all the inner elements are selected

### PR DESCRIPTION
**Pull request by somyarocketium**
If on mount all the elements inside a group are selected, selecting the group instead of the individual elements

### CODE REVIEW

**Review by prabhatkadam18 — COMMENTED** [review]
@somyarocketium I have not seen us pushing the files in `dist` folder as these are the build files I guess. Was this intentional?

File: src/AlignmentGuides.js [hunk 1]
```diff
@@ -127,12 +111,22 @@ class AlignmentGuides extends Component {
...
 						if (boxes.hasOwnProperty(box) && groupArray.includes(boxes?.[box]?.metadata?.captionIndex)) {
 							selections.push(boxes[box]);
+							if (boxes[box].active !== true) {
+								allElementsInsideGroupAreSelected = false;
+							}
 						}
 					}
+					if (allElementsInsideGroupAreSelected) {
+						selections.map(sel => {
+							activeBoxes.splice(sel);
```
[the remark this answers is not in the document]
Comment (somyarocketium) [reply src/AlignmentGuides.js:126]: Hey @prabhatkadam18 we are doing Array.splice(start) itself, selections is an array of indexes only

File: src/AlignmentGuides.js [hunk 2]
```diff
@@ -127,12 +111,22 @@ class AlignmentGuides extends Component {
...
 					for (let box in boxes) {
 						if (boxes.hasOwnProperty(box) && groupArray.includes(boxes?.[box]?.metadata?.captionIndex)) {
 							selections.push(boxes[box]);
+							if (boxes[box].active !== true) {
+								allElementsInsideGroupAreSelected = false;
+							}
 						}
 					}
+					if (allElementsInsideGroupAreSelected) {
+						selections.map(sel => {
```
[the remark this answers is not in the document]
Comment (somyarocketium) [reply src/AlignmentGuides.js:125]: Thanks for the suggestion @prabhatkadam18 will update this.

**Comment by somyarocketium** [comment]
@prabhatkadam18 yes unfortunately default somehow the files came in automatically

**Comment by somyarocketium** [comment]
@prabhatkadam18 did the changes, please do have a look

**Comment by somyarocketium** [comment]
@prabhatkadam18 did all the changes, can you review the same.

**Review by prabhatkadam18 — APPROVED** [review]

**Pull request merged**